### PR TITLE
gt, lt, 

### LINKED ISSUE #176: Bad error message for gt validation
I am using the latest pydantic release and I came here to let you know that there are problems with the validation error messages related to "gt" and "lt".

I have:

    ZeroOrMore: conint = conint(gt=-1)

    class Config(Pydantic):
        max_file_size: ZeroOrMore = 0

I define `ZeroOrMore` with `gt=-1` because zero must be allowed and the nearest lower integer is -1. And this much is correct, inasmuch the value -1 causes a validation error:

    max_file_size:
      size less than minimum allowed: -1 (error_type=ValueError track=ConstrainedIntValue)

But there are 2 problems in this error message:

1. The value printed should be 0, not -1, because 0 is the "minimum allowed".
2. The error message says "size". But this is not a size. Who said anything about sizes? The error message should simply read, "Lower than the minimum 0.".

This is related to #171 and #174.

### PR DESCRIPTION
**Pull request by samuelcolvin**
fix #176

### CODE REVIEW

File: tests/test_types.py [hunk 1]
```diff
@@ -836,3 +836,27 @@ class Model(BaseModel):
         assert exc_info.value.flatten_errors() == result
     else:
         assert Model(foo=value).foo == result
+
+
+def test_gt():
```
Comment (Gr1N) [inline tests/test_types.py]: maybe better to name `test_number_gt` and `test_number_lt`?

File: tests/test_types.py [hunk 2]
```diff
@@ -836,3 +836,27 @@ class Model(BaseModel):
...
+    class Model(BaseModel):
+        a: conint(gt=-1) = 0
+
+    assert Model(a=0).dict() == {'a': 0}
+    with pytest.raises(ValidationError) as exc_info:
+        Model(a=-1)
+    assert (
+        'ensure this value is greater than -1 (type=value_error.number.gt; limit_value=-1)'
```
Comment (Gr1N) [inline tests/test_types.py]: why you want to use string representation of errors not `.flatten_errors()` like in all other tests?

Comment (samuelcolvin) [reply tests/test_types.py]: I was just doing the quickest thing, doing it various different ways in tests is a good thing.

**Comment by Gr1N** [comment]
👍

**Pull request merged**